modules/indexing/src/test/java/org/apache/ignite/sqltests/CheckWarnJoinPartitionedTables.java

```diff
@@ -46,41 +59,32 @@
     }
 
     /** {@inheritDoc} */
-    @Override protected void afterTest() throws Exception {
+    @Override protected void beforeTest() throws Exception {
```

Comment:
It is like a good practive to stop the grid after test.
Please, revert this or stop grid after all tests.